### PR TITLE
Add MySQL dump token

### PR DESCRIPTION
**Pull request by mclmax**
This diff adds Jacob's MySQL token to Canarytokens.org.

Notable things for this token is that it introduces a new
listen channel on which we alert on.

A Twisted service listens on port 3306 and responds with MySQL
server info. This is needed in order for the Replica flow to trigger
on the MySQL client.

This token currently grabs the attacker's IP, hostname and MySQL locale.

### CODE REVIEW

File: settings.py [hunk 1]
```diff
@@ -90,8 +93,8 @@
     if not hasattr(settingsmodule, 'PROTOCOL'):
         setattr(settingsmodule, 'PROTOCOL', 'http')
 
-if WEB_IMAGE_UPLOAD_PATH and not os.path.exists(WEB_IMAGE_UPLOAD_PATH):
-    os.mkdir(WEB_IMAGE_UPLOAD_PATH)
+# if WEB_IMAGE_UPLOAD_PATH and not os.path.exists(WEB_IMAGE_UPLOAD_PATH):
```
Comment (nickrohrbs) [inline settings.py:96]: Why are we commenting this out?

File: templates/generate_new.html [hunk 2]
```diff
@@ -115,7 +115,8 @@ <h3 class="text-muted">
...
                     <li data-type="cloned_website" data-memo-placeholder="Cloned website token for https://thinkst.com"><a href="#" class="icon icon-clonedsite"><span class="title">Cloned Website</span><div class="explanation">Trigger an alert when your website is cloned</div></a></li>
-                    <li data-type="sql_server"     data-memo-placeholder="SELET SQL Server token on SQL01/CreditCards"><a href="#" class="icon icon-sqlserver"><span class="title">SQL Server</span><div class="explanation">Get alerted when MS SQL Server databases are accessed</div></a></li>
+                    <li data-type="sql_server"     data-memo-placeholder="SELECT SQL Server token on SQL01/CreditCards"><a href="#" class="icon icon-sqlserver"><span class="title">SQL Server</span><div class="explanation">Get alerted when MS SQL Server databases are accessed</div></a></li>
+                    <li data-type="my_sql"         data-memo-placeholder="MySQL token in payments_db_dump.sql on SRV01"><a href="#" class="icon icon-mysql"><span class="title">MySQL</span><div class="explanation">Get alerted when a MySQL dump is loaded</div></a></li>
```
Comment (nickrohrbs) [inline templates/generate_new.html:119]: Maybe `MySQL Dump` instead of just `MySQL`

File: templates/generate_new.html [hunk 3]
```diff
@@ -538,6 +539,59 @@ <h3>Your SQL Server token is active!</h3>
...
+
+<button class="btn btn-success btn-clipboard tooltip" type="button" data-clipboard-target="#mysql-statement-decoded">
+  <img src="/resources/clippy.svg" alt="Copy to clipboard">
+</button>
+                    </div>
+                  </div>
+                  <h4 style="margin-top: 10px;">OR</h4>
+                  <p class="mysql-text">2.) Download a (pseudo) random MySQL dump with a token already embedded in it</p>
+                  <div class="form-control">
+                    <a class="btn btn-large btn-success mysqldownload" data-fmt="my_sql">Download a My SQL Dump file</a>
```
Comment (nickrohrbs) [inline templates/generate_new.html:578]: MySQL*

File: settings.py (hunk 1 above)
Comment (mclmax) [reply settings.py:96]: ah damn yes! was erroring out when run locally

**Review by jayjb — APPROVED** [review]

**Pull request merged**